tf/admin/cloud-storage.tf

```diff
@@ -0,0 +1,9 @@
+resource "google_storage_bucket" "terraform_state" {
+  name          = "${var.GOOGLE_CLOUD_TF_STATE_BUCKET_NAME}"
+  location      = var.GOOGLE_CLOUD_REGION
+  force_destroy = true
```

Comment:
During development this works, but consider setting `force_destroy` to false once this is ready for production.

Reply:
good catch - i copied this from the src-archive bucket where cloud function code is stored, which i do actually want to force_destroy